composer.json

```diff
@@ -25,12 +25,12 @@
     },
     "require-dev": {
         "symfony-cmf/testing": "^1.3",
-        "matthiasnoback/symfony-dependency-injection-test": "~0.7,>=0.7.4",
+        "matthiasnoback/symfony-dependency-injection-test": "^0.7.4",
         "matthiasnoback/symfony-config-test": "^1.2.3",
-        "sonata-project/block-bundle": ">=2.2.8",
-        "sonata-project/core-bundle": ">=2.2.3",
+        "sonata-project/block-bundle": "^2.2.8",
+        "sonata-project/core-bundle": "^2.2.3",
         "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
-        "symfony-cmf/tree-browser-bundle": ">=1.0.0-RC1",
+        "symfony-cmf/tree-browser-bundle": "^1.0.0|^2.0.0",
```

Comment:
do we really want this? TreeBrowserBundle 2.0 cannot be included in the CMF 1.3 release (it will include TreeBrowserBundle 1.2 IIRC).